common/datasets/tedlium2/corpus.py

```diff
@@ -93,3 +93,39 @@ def get_stm_dict(output_prefix: str = "datasets") -> Dict[str, tk.Path]:
     :return:
     """
     return download_data_dict(output_prefix=output_prefix).stm
+
+
+def get_ogg_zip_dict(
+    subdir_prefix: str = "datasets",
+    returnn_python_exe: Optional[tk.Path] = None,
+    returnn_root: Optional[tk.Path] = None,
+    bliss_to_ogg_job_rqmt: Optional[Dict[str, Any]] = None,
+) -> Dict[str, tk.Path]:
+    """
+    Get a dictionary containing the paths to the ogg_zip for each corpus part.
+
+    No outputs will be registered.
+
+    :param str subdir_prefix: dir name prefix for aliases and outputs
+    :param tk.Path returnn_python_exe: path to returnn python executable
+    :param tk.Path returnn_root: python to returnn root
+    :param dict bliss_to_ogg_job_rqmt: rqmt for bliss to ogg job
+    :return: dictionary with ogg zip paths for each corpus (train, dev, test)
+    """
+    from i6_core.returnn.oggzip import BlissToOggZipJob
+
+    ogg_zip_dict = {}
+    bliss_corpus_dict = get_bliss_corpus_dict(audio_format="wav", output_prefix=subdir_prefix)
+    for name, bliss_corpus in bliss_corpus_dict.items():
+        ogg_zip_job = BlissToOggZipJob(
+            bliss_corpus,
+            no_conversion=False,
```

Comment:
```suggestion
            no_conversion=False,  # cannot be used for corpus with multiple segments per recording
```

Reply:
yes we can add this. note that an assert statement is also added to the bliss-to-ogg script now so it will crash anw.

Reply:
Yes, I saw that, that's good :+1: I think the comment still helps to make it clear why this is the case.